gcs/pom.xml

```diff
@@ -223,6 +223,14 @@
               </transformers>
               <filters>
                 <!-- minimizeJar breaks all the dynamic load hacks; add them explicitly -->
+                <filter>
```

Comment:
Give conscrypt is a signed jar, it was giving a "Exception in thread "main" java.lang.SecurityException: Invalid signature file digest for Manifest main attributes" when trying to create an uber shaded jar.
Thus removing all SF, DSA etc file to create unsigned jar. Refer: https://stackoverflow.com/questions/999489/invalid-signature-file-when-attempting-to-run-a-jar/6743609#6743609